packages/bolt-connection/src/bolt/bolt-protocol-v1.js

```diff
@@ -391,15 +468,15 @@ export default class BoltProtocol {
       this.packable(messageStruct)()
 
       this._chunker.messageBoundary()
-
       if (flush) {
         this._chunker.flush()
       }
     }
   }
 
-  isLastMessageLogin () {
-    return this._lastMessageSignature === 0x01
+  isLastMessageLogon () {
+    return this._lastMessageSignature === 0x01 ||
```

Comment:
For my own understaning, Could we not use modify request-message.js a little and use the consts there to do the checking to remove these literals?